library/oneview_server_hardware.py

```diff
@@ -73,17 +78,42 @@
 
 - name: Ensure that the Server Hardware is present and is inserted in the desired scopes
```

Comment:
You can add same comment here as well that its supported only till API 500

Reply:
Its in the examples as part of the documentation.
Comment is not required as we have added api_version.